core/plugins/org.fusesource.ide.foundation.core/src/org/fusesource/ide/foundation/core/util/OnlineVersionMapper.java

```diff
@@ -20,7 +20,7 @@
 
 public abstract class OnlineVersionMapper {
 	
-	protected static final String BASE_REPO_CONFIG_URI = "https://raw.githubusercontent.com/jbosstools/jbosstools-fuse/master/configuration/";
+	protected static final String BASE_REPO_CONFIG_URI = "https://raw.githubusercontent.com/lhein/jbosstools-fuse/FUSETOOLS-3387/configuration/";
```

Comment:
Is this `lhein` hardcoded repo intended?

Reply:
yes, for the PR we use our own fork and branch to have the tests fully running. after the merge we switch back to the original repo.